tests/models/test_tags.py

```diff
@@ -0,0 +1,88 @@
+from pyoffers.models.tags import TagRelation
+
+from pyoffers.models import Tag
+
+CASSETTE_NAME = 'tag'
+
+
+def test_create_success(tag):
+    assert isinstance(tag, Tag)
+    assert tag.name == tag.name
```

Comment:
Why to check attribute to equality with the same attribute?

Reply:
There was actual name there first, then I changed it when adjusting tests. I think this check is not necessary here at all, will remove.